projects/ad_quadmxfe1_ebz/common/ad_quadmxfe1_ebz_bd.tcl

```diff
@@ -336,7 +336,15 @@ for {set i 0}  {$i < $RX_NUM_OF_LINKS} {incr i} {
   }
 }
 
-ad_xcvrcon  util_mxfe_xcvr axi_mxfe_rx_xcvr axi_mxfe_rx_jesd $lane_map {} rx_device_clk
+if {$RX_NUM_OF_LANES != $MAX_RX_LANES} {
+  for {set i 0}  {$i < $MAX_RX_LANES} {incr i} {
+    if {$i ni $lane_map} {
+      lappend lane_map $i
+    }
+  }
+}
```

Comment:
This padding should be part of the ad_xcvrcon  procedure, this way you don't have to repeat it for every link.

Reply:
Created a new pull request for ad_xcvrcon: https://github.com/analogdevicesinc/hdl/pull/892